pdi-engine-api/impl/serializers/pom.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ ******************************************************************************
+  ~
+  ~  Pentaho Data Integration
+  ~
+  ~  Copyright (C) 2002-2017 by Pentaho : http://www.pentaho.com
+  ~
+  ~  *******************************************************************************
+  ~  Licensed under the Apache License, Version 2.0 (the "License"); you may not use
+  ~  this file except in compliance with the License. You may obtain a copy of the
+  ~  License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~  Unless required by applicable law or agreed to in writing, software
+  ~  distributed under the License is distributed on an "AS IS" BASIS,
+  ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~  See the License for the specific language governing permissions and
+  ~  limitations under the License.
+  ~
+  ~ ******************************************************************************
+  ~
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <artifactId>pdi-engine-impl</artifactId>
+    <groupId>org.pentaho</groupId>
+    <version>7.1-SNAPSHOT</version>
+  </parent>
+  <modelVersion>4.0.0</modelVersion>
+  <packaging>bundle</packaging>
+
+  <artifactId>pdi-engine-serializers</artifactId>
+
+  <properties>
+    <jackson.version>2.5.4</jackson.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.pentaho</groupId>
+      <artifactId>pdi-engine-api</artifactId>
+      <version>${project.version}</version>
+    </dependency>
+    <dependency>
+      <groupId>org.pentaho</groupId>
```

Comment:
Dependency does not exists collapsed with 'pdi-engine-api' in PR https://github.com/pentaho/pentaho-kettle/pull/3526